Gemfile

```diff
@@ -1,16 +1,19 @@
 source 'https://rubygems.org'
 gemspec
 
-gem 'debugger',      '~> 1.6.6'
+gem 'byebug', '~> 3.5.1' if RUBY_VERSION > '2'
 gem 'yard',          '~> 0.8.5'
 gem 'rake',          '~> 0.9'
 gem 'awesome_print', '~> 1.1.0'
 gem 'redcarpet'
 
-gem 'representors', git: 'https://www.github.com/mdsol/crichton-representors.git', tag: 'v0.0.1.pre'
+gem 'representors', git: 'https://www.github.com/mdsol-share/representors.git', branch: '0-0-stable'
```

Comment:
I think this should be on a tag of a sha.  Farscape is a critical part of our tools now, and it shouldn't have dependencies are moving targets.  It should be dependent on gems that which versions (tag/sha) that are none to work.